package.json

```diff
@@ -25,17 +26,21 @@
   "license": "MIT",
   "dependencies": {
     "@actions/core": "1.6.0",
-    "@actions/github": "2.2.0",
+    "@actions/github": "^6.0.0",
     "@slack/web-api": "^7.0.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.24.7",
```

Comment:
Ignorant question: Are we having to use babel and babel-jest because our code still targets node 12 or ES6?

https://github.com/npm/action-deploy/blob/92d23e286997461e72841fdb7ab223e77c49c0cb/action.yml#L51

https://github.com/npm/action-deploy/blob/92d23e286997461e72841fdb7ab223e77c49c0cb/tsconfig.json#L3

We're getting warnings on deploy that we should move our action to Node 20:

`The following actions uses node12 which is deprecated and will be forced to run on node16: npm/action-deploy@v2.4.0. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/`

`Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, npm/action-deploy@v2.4.0. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.`

https://github.com/npm/couchdb-sync/actions/runs/9594729076

Reply:
yes! you were spot on with this, was able to remove babel by targeting ES2020 in tsconfig